package.json

```diff
@@ -23,7 +23,6 @@
     "animate.css": "^3.5.2",
     "bootstrap": "^3.3.7",
     "bootstrap-datepicker": "^1.6.4",
-    "bootstrap-datetimepicker": "0.0.7",
```

Comment:
Duplicate changes from #3334.

Reply:
They are dependent of these. I was not sure who to do a merge request on a merge request